packages/react/src/components/Tag/InteractiveTag.stories.js

```diff
@@ -0,0 +1,305 @@
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const Selectable = () => {
+  return (
+    <>
+      <SelectableTag className="some-class" title="Clear Filter">
+        {'Tag content'}
+      </SelectableTag>
+      <SelectableTag
+        renderIcon={Asleep}
+        className="some-class"
+        title="Clear Filter">
+        {'Tag content'}
+      </SelectableTag>
+    </>
+  );
+};
+
+Selectable.argTypes = {
+  as: {
+    table: {
+      disable: true,
+    },
+  },
+  type: {
+    table: {
+      disable: true,
+    },
+  },
+  filter: {
+    table: {
+      disable: true,
+    },
+  },
+  onClose: {
+    table: {
+      disable: true,
+    },
+  },
+  selected: {
+    control: 'false',
+    description: 'Specify the state of the selectable tag.',
+  },
+  title: {
+    table: {
+      disable: true,
+    },
+  },
+};
+
+export const Operational = () => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <>
+      <div style={{ marginBottom: '1rem' }}>
+        <OperationalTag type="red" className="some-class" title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+        <OperationalTag
+          type="magenta"
+          className="some-class"
+          title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+        <OperationalTag
+          type="purple"
+          className="some-class"
+          title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+        <OperationalTag type="blue" className="some-class" title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+        <OperationalTag type="cyan" className="some-class" title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+        <OperationalTag type="teal" className="some-class" title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+        <OperationalTag
+          type="green"
+          className="some-class"
+          title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+        <OperationalTag type="gray" className="some-class" title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+        <OperationalTag
+          type="cool-gray"
+          className="some-class"
+          title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+        <OperationalTag
+          type="warm-gray"
+          className="some-class"
+          title="Clear Filter">
+          {'Tag content'}
+        </OperationalTag>
+      </div>
+
+      <h4>Interactive examples</h4>
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'flex-start',
+          marginTop: '0.5rem',
+        }}>
+        <Tooltip label="View more" align="bottom">
+          <OperationalTag className="some-class" title="Clear Filter">
+            {'Tag content'}
+          </OperationalTag>
+        </Tooltip>
+
+        <Toggletip>
```

Comment:
The `Toggletip` wrapper seems to be adding an extra tab stop here

Reply:
@tw15egan 
To fix that I would have to place a `tabIndex={-1}` 
```
<ToggletipButton label="Additional information" tabIndex={-1}>
```

But to do that I would have to add the `...rest` to `ToggletipButton`. Is that cool?

![Screenshot 2024-02-23 at 13 48 34](https://github.com/carbon-design-system/carbon/assets/52183462/c0b78c6b-0030-49a9-9912-ab041d931874)

Reply:
I think that's fine, [I just had to add ](https://github.com/carbon-design-system/carbon/pull/15792/files#diff-aff3fc17c1be744759ed294411fed5beccce2582e3eb2c39565e7bdddfc3a30a)`...rest` to `Toggletip` as well to handle some of the `Slug` tests

Reply:
However, that brings up another issue in this scenario; you've got a button inside a button, which I believe is an a11y violation. Is there a way we could change what either the `ToggletipButton` or `OperationalTag` is rendered as using an `as` prop or something?

Reply:
Humm, good observation. I'll check it out how to approach that.

Reply:
That makes sense, but now the `aria` roles seem invalid since it's no longer a button. This isn't necessarily a blocker for merging, but it might be worth creating a follow-up issue if using `Toggletip` to wrap the `OperationTag` is one of the intended use cases. The operation tag itself looks great 👍